tutorials/nnvm/from_tensorflow.py

```diff
@@ -0,0 +1,254 @@
+"""
+Compile Tensorflow Models
+====================
+This article is an introductory tutorial to deploy tensorflow models with NNVM.
+
+For us to begin with, tensorflow module is required to be installed.
+
+A quick solution is to install tensorlfow from
+
+https://www.tensorflow.org/install/install_sources
+"""
+
+import nnvm
+import tvm
+import numpy as np
+import os.path
+import re
+
+# Tensorflow imports
+import tensorflow as tf
+from tensorflow.core.framework import graph_pb2
+from tensorflow.python.framework import dtypes
+from tensorflow.python.framework import tensor_util
+
+
+repo_base = 'https://github.com/srkreddy1238/dmlc_data/raw/master/models/tensorflow/InceptionV1/'
```

Comment:
put data into dmlc/web-data

Reply:
https://github.com/dmlc/web-data/pull/74
PR for support files @ web-data.

I will update the above paths after this merge.

Reply:
data paths changed to dmlc/web-data.